scipy/spatial/transform/_rotation.pyx

```diff
@@ -1349,6 +1370,12 @@ cdef class Rotation:
         -------
         quat : `numpy.ndarray`, shape (4,) or (N, 4)
             Shape depends on shape of inputs used for initialization.
+        canonical : `bool`, default False
```

Comment:
This needs to be in `Parameters` not `Returns`.

Reply:
Done